src/Player/PlayerCard.js

```diff
@@ -40,7 +40,6 @@ const PlayerCard = props => {
   // just bring in ramda for utility fn's.
   // TODO: Separate props from state here.
   // TODO: Pass the player state to the `onEditClick` function passed as a prop.
-  const onEditClick = props.onEditClick ? props.onEditClick : function noop() {};
```

Comment:
Maybe we should add the check for props back in, OR make the `onEditClick` prop a required prop.